apps/web/src/ee/clerk/pages/OrganizationListPage.tsx

```diff
@@ -26,8 +26,8 @@ export default function OrganizationListPage() {
         }}
         hidePersonal
         skipInvitationScreen
-        afterSelectOrganizationUrl={ROUTES.GET_STARTED}
-        afterCreateOrganizationUrl={ROUTES.AUTH_APPLICATION}
+        afterSelectOrganizationUrl={ROUTES.WORKFLOWS}
+        afterCreateOrganizationUrl={ROUTES.WORKFLOWS}
```

Comment:
these should never happen as the sign-up process will happen through the new dashboard